chat-widget/src/components/livechatwidget/common/reconnectChatHelper.ts

```diff
@@ -132,4 +39,4 @@ const hasReconnectId = (reconnectAvailabilityResponse: IReconnectChatContext) =>
     return reconnectAvailabilityResponse && !isNullOrUndefined(reconnectAvailabilityResponse.reconnectId);
 };
 
-export { handleChatReconnect, isReconnectEnabled, isPersistentEnabled, getChatReconnectContext };
\ No newline at end of file
+export { redirectPage, isReconnectEnabled, isPersistentEnabled, hasReconnectId };
```

Comment:
my suggestion is to use : isReconnectIdPresent ?

Reply:
Renamed